src/online2bin/online2-tcp-nnet3-decode-faster_word_timestamps.cc

```diff
@@ -426,6 +431,9 @@ int main(int argc, char *argv[]) {
               istringstream iss(msg);
               vector<string> message_vector{istream_iterator<string>{iss}, istream_iterator<string>{}};
               std::string message = "";
+              std::string block_word_start = "";
```

Comment:
Could you try to find a better name for this variable if possible? I kind of don't understand its purpose without any further explanation. Especially it being a string makes it even more unclear (I know why it is a string, but let's make it clearer for new readers by improving the naming). In situations when you just cannot find a better name (I think there's not many of such) leave a comment at least.